daisy_workflows/export/export_disk.sh

```diff
@@ -13,20 +13,68 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+BYTES_1GB=1073741824
 URL="http://metadata/computeMetadata/v1/instance/attributes"
 GCS_PATH=$(curl -f -H Metadata-Flavor:Google ${URL}/gcs-path)
 LICENSES=$(curl -f -H Metadata-Flavor:Google ${URL}/licenses)
+DISK_RESIZING_MON=$(curl -f -H Metadata-Flavor:Google ${URL}/resizing-script-name)
 
-echo "GCEExport: Running export tool."
```

Comment:
Why remove this?

Reply:
It's replaced by multiple other messages because there are more detailed steps now. The most similar one is "GCEExport: Exporting image to local by gce_export tool..." at line 59

Reply:
Having an intro message that export is starting is still useful. Exporting to local is a single step of the export process and not equivalent of the whole process

Reply:
The thing is, it's not started at this point of time. The 1st message to see would be:
"GCEExport: Initializing buffer disk for gce_export output..."
I understand you are talking about a message for the whole exporting thing. That exists as "step disk-export is running..." in daisy log.